config/storybook/decorators.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Grid, Paper, Table, TableRow, TableCell, TableHead, TableBody } from "@material-ui/core";
```

Comment:
Please remove `material-ui` from this project

Reply:
removed